src/azure-account.api.d.ts

```diff
@@ -27,8 +27,12 @@ export interface AzureAccount {
 	createCloudShell(os: 'Linux' | 'Windows'): CloudShell;
 }
 
+export interface AzureAccountEnvironment extends Environment {
+	azureStackApiProfile?: boolean;
```

Comment:
What is an Api Profile? I think `isAzureStack` is clearer to me, but I'm not sure if I'm missing additional context

Reply:
"azure.target_azurestack_api_profile": true is the parameter we set in user setting. That is why Bala wants to use it as the name. @bganapa, do you think isAzureStack would be better for us. Or you still want to keep this one. If so, maybe you can jump in to give a more detailed explaination to Rachel about your thought.

Reply:
azureStackApiProfile is more appropriate here. APi Profile is a map of supported Resource Types and api versions.
More details here.
https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-version-profiles?view=azs-2008
Our customers access all the tools and SDKs through API Profile. For example, in Azure CLI, to work against the azurestack, the customer does the following
az cloud update --profile 2019-03-01-hybrid
Note that there can be multiple AuzreStack API Profiles supported at the same time. In the last few years , we have only one API profile active(supported). So we decided to keep it simple for VScode and made it boolean. 

Also the customer can work against AzureCloud or MoonCake with the same AzureStack API Profile. For example, if a customer wants to develop an App that works in AzureStack as well as Azure, we recommend using the API Profile SDKs instead of using two different SDKs (one for Azure and another for AzureStack)

Hope that naming makes sense